playbooks/DeploySDDCMgr.yml

```diff
@@ -0,0 +1,178 @@
+
+                                 Network IPv4 Address: {{ Nested_SDDCMgr.Component.Appliance.Address.IPv4.Address }}
+                                      Network Netmask: {{ (Nested_SDDCMgr.Component.Appliance.Address.IPv4.Address + '/' + Nested_SDDCMgr.Component.Appliance.Address.IPv4.Prefix) | ansible.utils.ipaddr('netmask') }}
+                                      Network Gateway: {{ Nested_SDDCMgr.Component.Appliance.Address.IPv4.Gateway }}
+
+                                   Network DNS Server: {{ Common.DNS.Server1.IPv4 }}
+                                   Network DNS Domain: {{ Common.DNS.Domain }}
+
+                                    SDDCMgr Root Password: {{ Nested_SDDCMgr.Credential.root.Password }}
+                                   SDDCMgr admin Password: {{ Nested_SDDCMgr.Credential.admin.Password }}
+                                   SDDCMgr local Password: {{ Nested_SDDCMgr.Credential.local.Password }}
+                                  SDDCMgr backup Password: {{ Nested_SDDCMgr.Credential.backup.Password }}
+                                     SDDCMgr vcf Password: {{ Nested_SDDCMgr.Credential.vcf.Password }}
+                                      SDDCMgr FIPS_Enable: {{ Nested_SDDCMgr.Component.Appliance.FIPSEnable }}
+
+                                       SDDCMgr Media Path: {{ Deploy.Software.SDDCMgr.Directory }}
+                                       SDDCMgr Media File: {{ Deploy.Software.SDDCMgr.File }}
+
+          =================================================================================================
+      when:
+        - DEBUG.DisplayVariables
+
+    - name: Check if SDDCMgr is already installed
+      ansible.builtin.uri:
+        url: https://{{ Nested_SDDCMgr.Component.Appliance.FQDN }}
+        validate_certs: "{{ Common.PKI.ValidateCerts }}"
+        timeout: 5
+      register: sddcmgr_check
+      ignore_errors: true
+      when:
+        - Deploy.Product.SDDCMgr.Deploy
+
+    - name: Deploy SDDCMgr from OVA
+      community.vmware.vmware_deploy_ovf:
+        hostname: "{{ Target.FQDN }}"
+        username: "{{ Target.User }}"
+        password: "{{ Target.Password }}"
+        validate_certs: "{{ Common.PKI.ValidateCerts }}"
+        datastore: "{{ Target.Datastore }}"
+        allow_duplicates: false
+        # For vCenter deployment, VM Folder path must be the FULL path: /<DataCenter>/vm/<ParentFolder>/<ChildFolder>
+        folder: "{% if Target.Deployment == 'vCenter' %}/{{ Target.DataCenter }}/vm/{{ Target.VMFolder }}\
+                 {% else %}{{ Target.VMFolder }}{% endif %}"
+        datacenter: "{{ Target.DataCenter }}"
+        cluster: "{{ Target.Cluster }}"
+        name: "{{ Nested_SDDCMgr.Component.Appliance.VMName }}"
+        ovf: "{{ Deploy.Software.SDDCMgr.Directory }}/{{ Deploy.Software.SDDCMgr.File }}"
+        networks:
+          "Management": "{{ Target.PortGroup.Management.Name }}"
```

Comment:
I had to change this from "Management" to "Network 1" in order for the OVF deployment to start. Is it the same on your side?

Reply:
I did not and I ran a couple test runs before creating the PR.  looking over the ovf file I don't know why it ever worked like this.  File updated.